.eslintrc.js

```diff
@@ -1,13 +1,16 @@
+/**
+ * @type {import("eslint").Linter.Config}
+ */
 module.exports = {
   parser: "@typescript-eslint/parser",
   parserOptions: {
-    project: ["./tsconfig.json"],
+    project: true,
```

Comment:
huh! do we not have to figure out which config to use anymore?

Reply:
I guess less so? I still needed to use it in our monorepo, but having it true where you can is what they are aiming for: https://typescript-eslint.io/blog/parser-options-project-true